### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader:NVDA does not announce Column header name  with header reference.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7609]
**GitHubTags**: #SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#DesktopApp;#NVDA#Win32;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727ff9fe68421b004bf270cd509e4c5de165
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Additional Details:**
NVDA 2019.2
**Steps to Reproduce :**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster” .
5.Start NVDA and Observe the announcement.
**Actual:**
NVDA does not announce Column header name  with header reference.
**Expected:**
NVDA should announce the  column header name with the header reference. 
**User Impact:**
Screen reader User will not able to understand if there are normal content of the table or  header of the table.

**Attachment for Reference:**
[A11y_AsureDataStudio_Selectthedeploymentoption_Screenreader3.pptx](https://github.com/microsoft/azuredatastudio/files/3707600/A11y_AsureDataStudio_Selectthedeploymentoption_Screenreader3.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
this should have been fixed now, i am able to have narrator read the column name.
